BlueRSA.podspec

```diff
@@ -1,17 +1,17 @@
 Pod::Spec.new do |s|
 s.name        = "BlueRSA"
-s.version     = "1.0.201"
+s.version     = "1.0.202"
 s.summary     = "Swift cross-platform RSA crypto library using CommonCrypto/libcrypto via Package Manager."
 s.homepage    = "https://github.com/Kitura/BlueRSA"
 s.license     = { :type => "Apache License, Version 2.0" }
 s.author     = "IBM and the Kitura project authors"
 s.module_name  = 'CryptorRSA'
 s.swift_version = "5.0.2"
 s.requires_arc = true
-s.osx.deployment_target = "10.12"
-s.ios.deployment_target = "10.3"
-s.tvos.deployment_target = "10.3"
-s.watchos.deployment_target = "3.3"
+s.osx.deployment_target = "11.5"
+s.ios.deployment_target = "14.5"
+s.tvos.deployment_target = "14.5"
+s.watchos.deployment_target = "7.5"
```

Comment:
Thanks for checking the deployment targets!  Just wanted to confirm if these are the minimum versions that will work with these changes?

Reply:
If it's a concern...can get away with going with even older versions...

I've just gotten in the habit with Apple of trying to stay as up to date as possible.